db/migrate/20250204104514_add_auto_close_to_competitions_table.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAutoCloseToCompetitionsTable < ActiveRecord::Migration[7.2]
+  def change
+    add_column :Competitions, :auto_close_threshold, :integer, default: 0, null: false
```

Comment:
Why is `0` a good choice for a default value here, instead of just letting it be `null`? The latter seems like the perfect choice to express "I don't want to use any auto-close".

Reply:
We don't currently have a way of setting values to `NULL` in `InputNumber` in the competition form, so once you enter a numeric value there is no way to disable that field. I've started using 0 as a default to get around this, but I strongly agree `NULL` would be more appropriate - I'm happy to take a detour to make this change in the FormInputs element now that I'm a little more React literate

Reply:
Yay! But you gotta pay attention that some parts of the form _want_ to avoid `NULL` and instead explicitly set stuff to 0. So you cannot just straight-out interpret 0 as NULL in each case.

My suggestion here is to pass in an additional prop, happy to hop on a call and whip this up together because the wrapped form components are a bit tricky